src/modules/10-common/utils/testUtils.tsx

```diff
@@ -141,83 +142,92 @@ export const TestWrapper: React.FC<TestWrapperProps> = props => {
   // }, [path, pathParams, queryParams])
 
   return (
-    <StringsContext.Provider value={{ data: stringsData as any, getString }}>
-      <AppStoreContext.Provider
-        value={{
-          featureFlags: {
-            FEATURE_ENFORCEMENT_ENABLED: true
-          },
-          updateAppStore: () => void 0,
-          currentUserInfo: {},
-          ...defaultAppStoreValues
-        }}
-      >
-        <LicenseStoreContext.Provider
+    <Router history={history}>
+      <StringsContext.Provider value={{ data: stringsData as any, getString }}>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232308)
```suggestion
    <StringsContext.Provider value={{ data: stringsData as unknown, getString }}>
```